### DISCUSSION TITLE
What is the limit to which I can use this app without breaking TOS?

### DISCUSSION THREAD

**Discussion by Harsharma2308** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#404] [General]
How can I keep going to use this app without getting a ban? I read something like 100 requests. Want to know the exact number if someone knows with the source. 
This client is great and I am elated I can control the connected smart lights from my pc now even though the personal results don't seem to be workng currently.

**Comment by im-coder-lg** [comment]
Personal results work, all you gotta do is follow the steps [here](https://gist.github.com/im-coder-lg/23fc93cbf983b36b21641563ed4f3d4e).

**Reply by im-coder-lg** [reply]
I tested it personally.
